package.json

```diff
@@ -1085,7 +1085,7 @@
     "cockatiel": "^3.1.1",
     "command-exists": "^1.2.9",
     "find-process": "^1.4.7",
-    "fluid-framework": "^1.3.7",
+    "fluid-framework": "^1.3.3",
```

Comment:
Hmm.   Taking a second look, both of the alerts listed in the summary are complaining about transitive dependencies of `@fluidframework/azure-client`, not `fluid-framework`.  Would this change actually have any effect on those alerts?

Reply:
I have updated the description, so alert 1 is solved by this version change, can have a final verification after merging to main, and about the second alert currently "@fluidframework/azure-client does not have a latest released version with updated axios version", hence it is a known issue.